ownCloud/Client/Viewer/DisplayHostViewController.swift

```diff
@@ -19,6 +19,19 @@
 import UIKit
 import ownCloudSDK
 import ownCloudAppShared
+import CoreServices
+
+extension OCItem {
+	var isPlayable: Bool {
+		guard let mime = self.mimeType else { return false }
+
+		guard let uti = UTTypeCreatePreferredIdentifierForTag(kUTTagClassMIMEType, mime as CFString, nil) else {
+			return false
+		}
+
+		return UTTypeConformsTo(uti.takeUnretainedValue(), kUTTypeAudiovisualContent)
+	}
+}
```

Comment:
Nice new addition to the `OCItem` class.
We already have a `OCItem+Extension`. I think it makes sense to move this code into this class, because it is more discoverable there.

Reply:
Not sure, this is context specific extension.. so for me it was easier to have it closer to the context in which it is used

Reply:
I think this extension is also interesting for an upcoming shortcut extension. For that I the `OCItem+Extension` is a better place in my opinion.

Reply:
Moved to the extension you mentioned.